compiler/src/iree/compiler/Dialect/LinalgExt/Utils/IndexingUtils.cpp

```diff
@@ -83,10 +84,6 @@ void AttentionOpDetail::inferFromIndexingMaps(
 
 FailureOr<AttentionOpDetail>
 AttentionOpDetail::get(ArrayRef<AffineMap> indexingMaps) {
-  if (indexingMaps.size() != 4 && indexingMaps.size() != 6) {
```

Comment:
Can we come up with an updated checker/verifiier?

Reply:
Probably just `if (getIndexingMaps().size() != getNumOperands()` in AttentionOp::verify()